scripts/lib/CIME/SystemTests/ncr.py

```diff
@@ -1,111 +1,61 @@
 """
 Implementation of the CIME NCR test.  This class inherits from SystemTestsCommon
 
-Build two exectuables for this test, the first is a default build the
-second halves the number of tasks and runs two instances for each component
-Lay all of the components out concurrently
+Build two exectuables for this test:
+The first is a default build
+The second runs two instances for each component with the same total number of tasks,
+and runs each of them concurrently
```

Comment:
Given that (as you said) you haven't been able to fully test this, can you please add a comment here - and ideally also in the config_tests.xml file - noting that this is untested and may not be working quite right?